### PR TITLE
[imapsync] - parse exit status value and show it human readable in new column

### PR DESCRIPTION
**Pull request by feldsam**
Signed-off-by: Kristian Feldsam <feldsam@gmail.com>

### CODE REVIEW

File: data/web/js/site/user.js [hunk 1]
```diff
@@ -224,6 +225,11 @@ jQuery(function($){
             if (!item.last_run > 0) {
               item.last_run = lang.waiting;
             }
+            if (item.success == null) {
+              item.success = '-';
```
Comment (andryyy) [inline data/web/js/site/user.js:229]: Should this be "currently running" or similar?

Comment (andryyy) [reply data/web/js/site/user.js:229]: Hm, we have is_running, so that's ok, I guess.

Comment (feldsam) [reply data/web/js/site/user.js:229]: Hi, the "item.success" is set to null when a job enters running state or when a job is rescheduled.

**Comment by feldsam** [comment]
@andryyy just updated with parsing exit status functionality. This is much better version for user experience

**Comment by feldsam** [comment]
Screenshot 2021-08-24 at 11 07 51

**Comment by andryyy** [comment]
Staaaaging.

**Pull request closed without merging**